ansible/roles/netlogon/files/system-auth

```diff
@@ -2,9 +2,9 @@
 
 auth    [success=4 default=ignore] pam_unix.so try_first_pass nullok
 auth    [success=3 default=ignore] pam_policycache.so try_first_pass action=check
-auth    [success=ok default=die] pam_netauth.so try_first_pass
-auth    [success=1 default=ignore] pam_policycache.so try_first_pass action=update
+auth    [success=1 default=die] pam_netauth.so try_first_pass
 auth    required  pam_deny.so
+auth    [default=ignore] pam_policycache.so action=update
```

Comment:
Are the brackets still necessary if not space in the second column?
Bikeshedding, but everything else seems alright to me.
Unless we don't want to cache pam_unix returns? I don't understand if that's a concern at all.

Reply:
`pam_unix` is success=4, so it skips all of the NetAuth and `pam_cachepolicy` actions, those won't be cached.

The brackets are still necessary, the keywords like `required`, `sufficient`, etc, just expand out to specific settings in brackets.

Reply:
[4.1. Configuration file syntax](http://www.linux-pam.org/Linux-PAM-html/sag-configuration-file.html) for reference:

> N (an unsigned integer)
>    equivalent to ok with the side effect of jumping over the next N modules in the stack.

>  The third field, control, indicates the behavior of the PAM-API should the module fail to succeed in its authentication task. There are two types of syntax for this control field: the simple one has a single simple keyword; the more complicated one involves a square-bracketed selection of value=action pairs. 

(Side note: Files in `/etc/pam.d` do not have the first field, as it is implied by the filename.)